qiskit_ibm/ibm_backend.py

```diff
@@ -778,11 +773,12 @@ def __init__(
 
         Args:
             configuration: Backend configuration.
-            provider: IBM Quantum account provider
+            provider: IBM Quantum account provider.
             credentials: IBM Quantum credentials.
             api_client: IBM Quantum client used to communicate with the server.
         """
         super().__init__(configuration, provider, credentials, api_client)
+        self._provider = provider
```

Comment:
Not that it affects the functionality, but why initialize the provider twice (it's already set in the line above)?